### ISSUE TITLE
[A11y][MultiComboBox]: selected state of MultiComboBoxItem is not announced via screenreader

### ISSUE THREAD

**Issue by caroline-ferri** [UI5/webcomponents#12989]
### Bug Description

When the user expands the MultiComboBox the screenreader does not announced the MultiComboBoxItem's selected state (selected and non selected). 

### Affected Component

MultiComboBox

### Expected Behaviour

The selected state of the MultiComboBoxItem should be announced by the screen reader to convey the state to the user. 

### Isolated Example

https://js-sbcxezrf.stackblitz.io

### Steps to Reproduce

1. Navigate to isolated example (or [UI5 Webcomponents Sample](https://ui5.github.io/webcomponents/components/MultiComboBox/) with JAWS or NVDA screenreader
2. Expand the MultiComboBox 
3. Tab through the selected and non selected items
4. Notice that the selected state for both item types are not announced to the user
...

### Priority

Medium

### UI5 Web Components Version

2.18.1

### Browser

Chrome

### Operating System

MacOS

### Organization

SFSF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by hristop** [comment]
Hi @caroline-ferri ,

There was already such feature request which will be handled with FIORITECHP1-30915 so i will close this one. Please continue monitoring the internal BLI for updates and progress.

Best Regards,
Hristo
